winterfell/services/userService.js

```diff
@@ -77,3 +79,74 @@ module.exports.upsertUserValues = function(userId, keyValues, callback) {
     userValue.save(callback);
   });
 };
+
+/**
+ * Expected input object from idme:
+ * {
+ *  id
+ *  verified
+ *  affiliation
+ *  fname
+ *  lname
+ *  zip
+ *  uuid
+ *  email
+ * }
+ */
+module.exports.createNewUserFromIdMe = function(idmeInfo, callback) {
+  var newUser = {
+    firstname: idmeInfo.fname,
+    middlename: null,
+    lastname: idmeInfo.lname,
+    email: idmeInfo.email,
+    password: "fakepassword", // todo: generate a short random 8-char password
```

Comment:
Probably they should just have no password if they are using oAuth right? They will authenticate with oAuth and not password oAuth from then on..